docs/installation-and-operation/configuring-application-database.md

```diff
@@ -6,71 +6,129 @@ redirect_from:
 
 # Configuring the Metabase application database
 
-The application database is where Metabase stores information about users, saved questions, dashboards, and any other
-data needed to run the application. The default settings use an embedded H2 database, but this is configurable.
+The application database is where Metabase stores information about user accounts, questions, dashboards, and any other data needed to run the Metabase application.
 
-## Notes
+For production, we recommend using PostgreSQL as your application database.
 
-- Using Metabase with an H2 application database is not recommended for production deployments. For production
-  deployments, we highly recommend using PostgreSQL. If you decide to continue to use H2, please be sure to back up the database file regularly.
-- You cannot change the application database while the application is running. Connection configuration information is
-  read only once when the application starts up and will remain constant throughout the running of the application.
-- Metabase provides limited support for migrating from H2 to Postgres if you decide to upgrade to a more
-  production-ready database. See [Migrating from H2 to PostgreSQL](migrating-from-h2.md) for more details.
+- [PostgreSQL](#postgresql) (recommended for production)
+- [MySQL](#mysql-or-mariadb) (also works for production)
+- [H2](#h2-default) (default for local demos - AVOID in production)
 
-## [H2](https://www.h2database.com/) (default)
+Metabase will read the connection configuration information when the application starts up. You can't change the application database while the application is running.
 
-> **For production installations of Metabase we recommend that people [replace the H2 database with PostgreSQL](./migrating-from-h2.md)**. Postgres offers a greater degree of performance and reliability when Metabase is running with many users.
+## PostgreSQL
 
-To use the H2 database for your Metabase instance you don't need to do anything at all. When the application is first launched it will attempt to create a new H2 database in the same filesystem location the application is launched from.
+We recommend that you use [PostgreSQL](https://www.postgresql.org/) for your Metabase application database.
 
-You can see these database files from the terminal:
+You can use [environment variables](../configuring-metabase/environment-variables.md) to set a Postgres database as Metabase's application database. For example, the following commands tell Metabase to use a Postgres database as its application database:
 
-    ls metabase.*
+```sh
+export MB_DB_TYPE=postgres
+export MB_DB_DBNAME=metabase
+export MB_DB_PORT=5432
+export MB_DB_USER=<username>
+export MB_DB_PASS=<password>
+export MB_DB_HOST=localhost
+java -jar metabase.jar
+```
 
-You should see the following files:
+Metabase will not create a Postgres database for you. Example command to create the database:
+
+```sh
+createdb --encoding=UTF8 -e metabase
```

Comment:
Always seemed kind of weird to me that we give a psql cli command for postgres but SQL for mysql...